VMBackup/main/handle.py

```diff
@@ -79,7 +79,7 @@ def main():
         hutil.patching = MyPatching
         configSeqNo = -1
         hutil.try_parse_context(configSeqNo)
-        disable_event_logging = hutil.get_intvalue_from_configfile("disable_logging", 1)
+        disable_event_logging = hutil.get_intvalue_from_configfile("disable_logging", 0)
```

Comment:
just wondering if it is possible to add another setting to allow for in-process (without another thread) or out-of-process (with a separate thread) processing of logs. That could allow you to switch and perform more validations as necessary.